crates/ide/src/lib.rs

```diff
@@ -405,7 +405,30 @@ impl Analysis {
         self.with_db(|db| {
             symbol_index::world_symbols(db, query)
                 .into_iter() // xx: should we make this a par iter?
-                .filter_map(|s| s.def.try_to_nav(db))
```

Comment:
Oh no, I absolutely did not fix the regression from my refactor since I forgot to change this line 😆 
That should be
```diff
-                .filter_map(|s| s.def.try_to_nav(db))
+                .filter_map(|s| s.try_to_nav(db))
```